packages/util/test/deepCopy.test.ts

```diff
@@ -100,7 +99,19 @@ describe('deepExtend', () => {
       },
       { a: source }
     );
-    assert.deepEqual({ a: source }, target);
-    assert.strictEqual(source, target.a);
+    expect({ a: source }).to.deep.equal(target);
+    expect(source).to.equal(target.a);
+  });
+
+  it('does not extend property __proto__', () => {
+    const src = JSON.parse('{ "__proto__": { "polluted": "polluted" } }');
+    const a = {};
+    deepExtend(a, src);
+
+    // @ts-expect-error
```

Comment:
In my opinion, this usage of "ts-expect-error" doesn't quite match the spirit. It seems geared towards explicit invalid usage, and would for example allow to test input validation. The usage here is not testing an error case. I would recommend just declaring `a` as an object with arbitrary keys.

Reply:
Good point. I got lazy.